kanidmd/lib/src/access.rs

```diff
@@ -514,7 +511,18 @@ pub trait AccessControlsTransaction<'a> {
             }
             IdentType::User(u) => &u.entry,
         };
-        trace!(event = %se.ident, "Access check for search (filter) event");
+        info!(event = %se.ident, "Access check for search (filter) event");
+
+        match se.ident.access_scope() {
+            AccessScope::IdentityOnly | AccessScope::Synchronise => {
+                security_access!("identity access scope is not permitted to search");
```

Comment:
Does this need to be a double log? Why not just 

`"denied ❌ - identity access scope is not permitted to search"`